android/app/src/main/java/safetyfirst/androidapp/safetyfirstcontroller/MainActivity.java

```diff
@@ -145,6 +178,85 @@ public void onCheckedChanged(CompoundButton compoundButton, boolean b) {//Publis
         });
     }
 
+    @SuppressLint("NonConstantResourceId")
+    @Override
+    public boolean onNavigationItemSelected(@NonNull MenuItem item) {
```

Comment:
```suggestion
    public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
```
maybe a more explanatory variable name here

Reply:
Suggested change implemented.